packages/expo/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
should we update `update-release-content.sh` file?

Reply:
I'd like to remove it from all packages as it didn't bring any value recently but slows down development.
I'll show the list of files in the expo-datadog release before the first release to make sure I don't bring anything not needed.